program.cpp

```diff
@@ -1,40 +1,66 @@
-#include "helper.h"
+#include "includes.h"
 
 using namespace std;
 using std::vector;
 
 // To test txt configs, change to "config"
-#define CONFIG_DIR "json"
+#define CONFIG_DIR "config"
 
+// Method to find the game folder name to store in config class.
+string get_folder_name(string entryPath)
+{
+    ConfigData config;
+
+    unsigned lastPath = entryPath.find("\\", 8);
+    string dir = entryPath.substr(0, lastPath);
+    unsigned findBackSlash = dir.find("\\");
+    dir.replace(findBackSlash, 1, "/");
+
+    return dir;
+}
+
+// Method to find the config files in the games directory.
 vector<string> get_config_files(string dir)
 {
     vector<string> files;
-    for(const auto & entry : fs::directory_iterator(dir))
+
+    for (const auto & entry : fs::recursive_directory_iterator(dir))
     {
-        files.push_back(entry.path().string());
-        cout << entry.path().string() << endl;
+        if (entry.path().filename() == "config.txt")
+        {
+            std::cout << entry << '\n';
+            files.push_back(entry.path().string());
+            cout << entry.path().string() << endl;
+            string path = entry.path().string();
+        }
     }
+
     return files;
 }
 
+// Create configs vector from config files.
 vector<ConfigData> config_data_list()
-{
-    vector<string> files = get_config_files(CONFIG_DIR);
+{   
+    vector<string> files = get_config_files("./games");
```

Comment:
@studioant As it needs to access the /games folder before it can find the config file it needs to be separate from the CONFIG_DIR. The CONFIG_DIR is currently being used to compare config and json files.  I can create a new constant for the games folder string and pass that in.